scripts/test.sh

```diff
@@ -6,9 +6,13 @@ set -e
 export TEST_SCRIPT=1
 export NPM_CONFIG_PROGRESS=false
 
-TARGET="$1"
-CONFIGURATION="${2:-"Release"}"
-DESTINATION=
+TARGET=$1
+CONFIGURATION=${2:-Release}
+
+IOS_SIM_DEVICE=${IOS_SIM_DEVICE:-} # use preferentially, otherwise will be set and re-exported
+ios_sim_default_device_type=${IOS_SIM_DEVICE_TYPE:-iPhone 5s}
```

Comment:
No, not for setting variables. And for the default (`:-`) the end curly brace is the implicit end-quote. E.g.:
```bash
bob=${IOS_SIM_DEVICE_TYPE:-iPhone 5s}
echo "a - $bob - b"
```
Will get you `a - iPhone 5s - b`